### PR TITLE
Adds furday

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

https://cdn.discordapp.com/attachments/354087825936875521/788453905796825109/final_5fd8234f45b5c7009372f89e_20613.mp4

10% chance to execute mass_purrbation at roundstart.

image

## Why It's Good For The Game

Its not

## Changelog
:cl:
add: Furday
/:cl:

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -589,3 +589,11 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
...
 /datum/holiday/easter/getStationPrefix()
 	return pick("Fluffy","Bunny","Easter","Egg")
+
+/datum/holiday/furday
+	name = "Furday"
+	begin_day = 20
+	begin_month = DECEMBER
+
+/datum/holiday/furday/getStationPrefix()
+	return pick("Boop", "Snoot", "Furry", "Artist")
```
Comment (Archanial) [inline code/modules/holiday/holidays.dm:599]:
```suggestion

```

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/holiday/holidays.dm:599]: thank me later

**Comment by ToasterBan** [comment]
### **OwO**

**Comment by Archanial** [comment]
It doesn't even pass linters
smh smh my head

**Comment by ghost** [comment]
> It doesn't even pass linters
> smh smh my head

Linters is actually broken.

**Comment by R3pID-4207** [comment]
I am not a fan of this one

**Comment by Archanial** [comment]
> 
> 
> > It doesn't even pass linters
> > smh smh my head
> 
> Linters is actually broken.

THATS THE JOKE

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -589,3 +589,17 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
...
+
+/datum/holiday/furday/greet()
+	. = ..()
+	if(prob(10))
+		mass_purrbation()
+		log_game("Furday randomly mass-purbatted everyone.")
```
Comment (Archanial) [inline code/modules/holiday/holidays.dm:605]:
```suggestion

```

**Comment by ghost** [comment]
> > > It doesn't even pass linters
> > > smh smh my head
> > 
> > 
> > Linters is actually broken.
> 
> THATS THE JOKE

image0-17

**Comment by PowerfulBacon** [comment]
very important bcus 5 days til furday

**Comment by DatBoiTim** [comment]
Well we know where the priorities lie.

**Comment by Archanial** [comment]
bacon ohw (on his way) to get qwerty to merge another dumbass meme pr
tenor

**Comment by Blueturbo47** [comment]
All felinids get felinid disease?

**Comment by TeomanTheGreat** [comment]
Question am I allowed to slaughter felinids on this day?

**Comment by DatBoiTim** [comment]
> bacon ohw (on his way) to get qwerty to merge another dumbass meme pr
> tenor

Felinid Paywall 2: Electric Boogaloo

**Comment by PowerfulBacon** [comment]
> 
> 
> Question am I allowed to slaughter felinids on this day?

no you have to become a furry and do whatever they say

**Comment by TeomanTheGreat** [comment]
> > 
> > 
> > Question am I allowed to slaughter felinids on this day?
> 
> no you have to become a furry and do whatever they say

No

**Comment by Penwin0** [comment]
Im going to cut off your grubby little hands so you cant code no more

**Comment by qwertyquerty** [comment]
Get checks to pass and I merge

**Comment by ghost** [comment]
> Get checks to pass and I merge

Merge it.

**Comment by OceanFish1** [comment]
FURRY CODING

FURRY CODING

**Comment by ike709** [comment]
no fun allowed

**Pull request closed without merging**